packages/nx-go/src/go-package-graph/get-go-imports.go

```diff
@@ -0,0 +1,111 @@
+package main
```

Comment:
**issue:** We are missing a `go.mod` file to track the dependencies of the code.

Please add a `go.mod` file in the `packages/nx-go` folder.

We need to then add `"packages/nx-go/go.mod",` to https://github.com/nx-go/nx-go/blob/main/packages/nx-go/project.json#L29